pkg/image/imageversion.go

```diff
@@ -69,22 +76,53 @@ func (c *ConformanceImageVersion) Set(str string) error {
 	return nil
 }
 
-// Get retrieves the preset version if there is one, or queries client if the ConformanceImageVersion is set to `auto`.
+// Get retrieves the preset version if there is one, queries client if the ConformanceImageVersion is set to `auto`,
+// or finds the latest dev image published.
 // kubernetes.Interface.Discovery() provides ServerVersionInterface.
 // Don't require the entire kubernetes.Interface to simplify the required test mocks
-func (c *ConformanceImageVersion) Get(client discovery.ServerVersionInterface) (string, error) {
-	if *c == ConformanceImageVersionAuto {
+func (c *ConformanceImageVersion) Get(client discovery.ServerVersionInterface, latestURL string) (registry, version string, returnErr error) {
```

Comment:
nit: probably the named return values is insignificant here?

Reply:
The reason to add naming is to disambiguate the two string values; otherwise its not clear what a `string string error` values will indicate. Since those are named the error needs named too to compile.

Reply:
ah ok makes sense